travis/trigger-hyrax-docker.sh

```diff
@@ -16,8 +16,15 @@ git config --global user.email "npotter@opendap.org"
 # This will get the BES and libdap version numbers
 BES_SNAPSHOT=$(cat ./bes-snapshot)
 
-OLFS_SNAPSHOT_TAG="olfs-${OLFS_BUILD_VERSION} "`date "+%FT%T%z"`
-HYRAX_SNAPSHOT_TAG="hyrax-${HYRAX_BUILD_VERSION} "`date "+%FT%T%z"`
+test_deploy=""
+if [[ "$TRAVIS_BRANCH" == *"-test-deploy" ]]
+then
+  test_deploy=" test-deploy"
+fi
+
+
+OLFS_SNAPSHOT_TAG="olfs-${OLFS_BUILD_VERSION} "$(date "+%FT%T%z")"$test_deploy"
```

Comment:
I'm a little surprised that spaces are okay in docker tags...but it seems like that was already happening between the build version and the date, so 🤷 !

Reply:
The docker tags are computed by the `hyrax-docker/builds/build-rh8` script.